pkg/member/member_control.go

```diff
@@ -177,6 +177,9 @@ func getMemberPeerURL(configFile string, podName string) (string, error) {
 		return "", err
 	}
 	initAdPeerURL := config["initial-advertise-peer-urls"]
+	if initAdPeerURL == nil {
+		return "", fmt.Errorf("initial-advertise-peer-urls must be set in etcd config.", err)
```

Comment:
```suggestion
		return "", fmt.Errorf("initial-advertise-peer-urls must be set in etcd config.")
```

Reply:
there is no need to add a error as `err` is coming from previous statement and it is `nil` as well

Reply:
Instead of using `fmt.Errorf` you can just use errors.New as the error message is static.

Reply:
sure, thanks for the hint :+1: